src/vm/appdomain.hpp

```diff
@@ -3441,8 +3441,8 @@ class SystemDomain : public BaseDomain
 
     //****************************************************************************************
     // Methods used to get the callers module and hence assembly and app domain.
-    __declspec(deprecated("This method is deprecated, use the version that takes a StackCrawlMark instead"))
-    static Module* GetCallersModule(int skip);
+
+    static DEPRECATED_MSG("This method is deprecated, use the version that takes a StackCrawlMark instead")  Module* GetCallersModule(int skip);
```

Comment:
Is this method used anywhere? Better to just delete it.

Reply:
it is used in the same file: 
src/vm/appdomain.cpp:    Module* mod = GetCallersModule(stackMark, ppAppDomain);

Reply:
Too early.

Apparently, GetCallersModule has another overloaded version. I'll remove this variant.